src/selectors/__tests__/exportContext.ts

```diff
@@ -152,3 +152,16 @@ it('export multi-line thoughts as separate thoughts', () => {
       - Hello
       - world`)
 })
+
+it('exported as markdown', () => {
```

Comment:
This test is good.

We should also a have a test that ensure that metaprogramming attributes are not escaped.